bin/amazon-collector

```diff
@@ -54,7 +56,8 @@ TopologicalInventoryIngressApiClient.configure.host   = "#{ingress_api_uri.host}
 begin
   metrics = TopologicalInventory::Amazon::Collector::ApplicationMetrics.new(args[:metrics_port])
   if args[:config].nil?
-    collector = TopologicalInventory::Amazon::Collector.new(args[:source], args[:access_key_id], args[:secret_access_key], metrics)
+    collector = TopologicalInventory::Amazon::Collector.new(
+      args[:source], args[:access_key_id], args[:secret_access_key], args[:sub_account_role], metrics)
```

Comment:
@Ladas if there are more input args, it means that it should be changed in UI too, right?
and this `:sub_account_role` will be saved to `Endpoint.extra` jsonb (like Azure: tenant_id). 
Then also orchestrator should be updated to integrate this data to config map and source's digest.
And https://github.com/ManageIQ/topological_inventory-amazon/blob/master/lib/topological_inventory/amazon/collectors_pool.rb#L31 has to be updated too

Reply:
yes that is correct